docs/setup.md

```diff
@@ -0,0 +1,78 @@
+---
+Order: 30
+Title: Setup
+Description: Instructions how to setup the Cake.Issues.PullRequests.Tfs addin.
+---
+This page describes the different ways how the [Cake.Issues.PullRequests.Tfs addin] can be setup.
+
+# NTLM authentication
+
+:::{.alert .alert-info}
+NTLM authentication is only available for on-premise Team Foundation Server.
+:::
+
+To authenticate with NTLM you can use the [TfsAuthenticationNtlm] alias from the [Cake.Tfs addin].
+
+The user needs to have `Contribute to pull requests` permission for the specific repository to
+allow [Cake.Issues.PullRequests.Tfs addin] to post issues as comments to pull requests.
+
+# Basic authentication
+
+:::{.alert .alert-info}
+Basic authentication is only available for on-premise Team Foundation Server.
+:::
+
+To authenticate with basic authentication you can use the [TfsAuthenticationBasic] alias from the [Cake.Tfs addin] and 
+need to [Configure TFS to use Basic Authentication].
+
+The user needs to have `Contribute to pull requests` permission for the specific repository to
+allow [Cake.Issues.PullRequests.Tfs addin] to post issues as comments to pull requests.
+
+# Personal access token
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.